components/common/typescript.gni

```diff
@@ -45,7 +45,8 @@ brave_common_web_compile_inputs = [
 
   # brave-ui changes (perhaps a manual local link which wouldn't affect
   # //brave/package.json)
-  "//brave/node_modules/brave-ui/package.json",
+  "//brave/node_modules/@brave/brave-ui/package.json",
+  "//brave/node_modules/@brave/leo/package.json",
```

Comment:
Do we need to add Leo here? It seems kind of unrelated to the other changes? I also don't think we need to compile any Leo stuff with Typescript (but I could be wrong :laughing:)

Reply:
It's more to make sure that WebUIs get re-compiled with webpack when Leo changes are made. It's not typescript related. Granted, those changes would have to be made without changing the brave/package.json file. It would have to be an npm install without a save. But if we're doing that strategy for brave-ui then we should do it for leo IMO.

Reply:
Actually, I keep forgetting that _all_ dependency modules will be included in the generated *.d "dep" files, including files in node_modules, so if any of those change, and they're used in a target, then that target will be recompiled.

Reply:
removed them